src/wrappers/wrapper.ts

```diff
@@ -45,7 +45,7 @@ export class Wrapper {
   constructor() {
     this.maxLines(Infinity);
     this.textTrimming("ellipsis");
-    this.allowBreakingWords(true);
+    this.allowBreakingWords(false);
```

Comment:
We shouldn't change the default since it's a significant breaking change.

Reply:
Can we at least set this as this as the default when we update plottable? Line-wrapping naively in the middle of a word is pretty ugly.

Reply:
Yeah we can turn it on in Plottable by default

Reply:
Yeah, let's major rev.